Library/Homebrew/build.rb

```diff
@@ -235,12 +235,12 @@ def fixopt(formula)
   # BuildErrors are specific to build processes and not other
   # children, which is why we create the necessary state here
   # and not in Utils.safe_fork.
-  case error_hash["json_class"]
-  when "BuildError"
+  case e
```

Comment:
I think this indicates this may need a rebase?

Reply:
I'm not following this one – care to elaborate? (In case it's helpful, `json_class` is just the class as a [string](https://github.com/flori/json#more-examples), but using the actual class name allows sorbet to type `e` and avoid errors when using methods that are specific to the actual exception class).

Reply:
@dduugg I think @issyl0 renamed all the single letter variable usage but could be wrong here, hence my thinking this PR may need a `git rebase origin/master` to avoid failing after merge.

Reply:
I renamed single letter variables _where they were method parameter names_ (and by extension where they were used in the methods). The `e` is allowed here until we go down the route of "no single letter variables at all" which we might do but not right now. But I'd still prefer a slightly more descriptive "error"?

Reply:
Ah, gotcha. I believe this name is actually dictated by [Naming/RescuedExceptionsVariableName](https://docs.rubocop.org/rubocop/cops_naming.html#namingrescuedexceptionsvariablename).

Reply:
Ahh I see, let's keep it then.

Reply:
Thanks both! Happy to land as-is.

FYI @issyl0: this is the case the merge queue would handle. If I was correct here and merged this PR: it would break the `master` build.